pyrgg/utils.py

```diff
@@ -0,0 +1,61 @@
+# -*- coding: utf-8 -*-
+"""Pyrgg utility module."""
+
+
+def get_min_weight(weight_dic):
```

Comment:
I suggest combining the `get_min_weight` and `get_max_weight` functions to improve efficiency.